modules/cassandra/src/main/java/org/testcontainers/containers/delegate/CassandraDatabaseDelegate.java

```diff
@@ -0,0 +1,65 @@
+package org.testcontainers.containers.delegate;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.exceptions.DriverException;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.delegate.AbstractDatabaseDelegate;
+import org.testcontainers.exception.ConnectionCreationException;
+import org.testcontainers.ext.ScriptUtils;
+
+import static org.testcontainers.containers.CassandraContainer.CQL_PORT;
+
+/**
+ * Cassandra database delegate
+ *
+ * @author Eugeny Karpov
+ */
+@Slf4j
+public class CassandraDatabaseDelegate extends AbstractDatabaseDelegate<Session> {
+
+    private CassandraContainer container;
+
+    public CassandraDatabaseDelegate(CassandraContainer container) {
```

Comment:
please mark `container ` as `final` and add `@RequiredArgsConstructor`

Reply:
Oh, I forgot about Lombok. Done, thanks! :+1: